src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,53 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePageClick = (page: number) => {
+    if (page >= 1 && page <= totalPages) {
+      onPageChange(page);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <button
+          className="page-link"
+          onClick={() => handlePageClick(currentPage - 1)}
+          disabled={currentPage === 1}
```

Comment:
It's a good practice to avoid inline functions within the render method, especially for components that will re-render often. This is because a new function is created on every render, which can lead to performance issues on larger components. Consider moving this to a class method or using `useCallback` if converting to a functional component with hooks.